README.md

```diff
@@ -1,65 +1,69 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+> A basic "Hello Microverse" page that illustrate the use of tools and best practices for future Microverse projects.
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Linters
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link - coming soon]()
 
 ## Getting Started
 
```

Comment:
The README's "Getting Started" section doesn't show how to start using the project locally. Why don't you add some steps about how to start using the project to it so your README is more informative and professional?